src/Microsoft.Extensions.DiagnosticAdapter/project.json

```diff
@@ -1,6 +1,15 @@
 {
   "version": "1.0.0-*",
-  "description": "A DiagnosticListener implementation which provides duck-typing through generated proxy types.",
+  "description": "Microsoft extension adapter feature to extend DiagnosticListener. Contains DiagnosticListener extension methods and types to enable duck-typing based event handling through generated proxy types.\r\nCommonly used DiagnosticListener extension method:\r\nSystem.Diagnostics.SubscribeWithAdapter",
```

Comment:
The list of commonly used types doesn't list any commonly used types. It has some new syntax I think you invented 😄 I'd omit it.

Suggestion:
* `Contains extension methods that extend System.Diagnostics.DiagnosticListener, and enables duck-typing based event handling by using dynamically generated proxy types.`